syntax = "proto2";

package vm_scheduler.proto;
option go_package = "proto/agent_api";

import "structures/job.proto";
import "structures/task.proto";


message JobExecutionResult {
  required JobStatus status = 1;
  optional string result_url = 2;
}

message TaskExecutionResult {
  required TaskStatus status = 1;
  repeated JobExecutionResult job_results = 2;
}
